zebra-state/src/config.rs

```diff
@@ -15,28 +15,41 @@ pub struct Config {
     /// include private data that cannot be replicated from the network, such as
     /// wallet data.  That data is not handled by `zebra-state`.
     ///
-    /// Each network has a separate state, which is stored in "mainnet/state"
-    /// and "testnet/state" subdirectories.
+    /// Each state format version and network has a separate state.
+    /// These states are stored in `state/vN/mainnet` and `state/vN/mainnet` subdirectories,
+    /// underneath the `cache_dir` path.
```

Comment:
Optional:
```suggestion
    /// These states are stored in `state/vN/mainnet` and `state/vN/mainnet` subdirectories,
    /// underneath the `cache_dir` path, where `N` is the state format version.
```

Reply:
Hey, i added both suggestions, had to make a new one for this one as github was not allowing to commit both of them changing the same line.